.github/workflows/pull_request.yml

```diff
@@ -6,6 +6,23 @@ on:
     - master
 
 jobs:
+  changelog:
+    name: CHANGELOG
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
```suggestion
    - uses: actions/checkout@v2
```

:)

Reply:
`git diff --name-only $(git merge-base origin/master HEAD)` will actually fail if we use v2.

Reply:
https://github.com/allenai/allennlp/pull/4260/checks?sha=0b7d61bb9538094980769b1e34338909a7af778a

Reply:
I didn't dig into this too much because I was being lazy. But if there's an easy way to make it work with v2 I'm all ears

Reply:
Ohh :+1:  Do you think it's worth it to add a comment?

Reply:
Definitely!